src/pages/library/dataBrowser-utils.test.js

```diff
@@ -31,17 +34,53 @@ describe('dataBrowser-utils', () => {
   })
 
   it('finds the correct data use policy to display if it exists', () => {
-    const { getByText } = render(h(DatasetReleasePolicyDisplayInformation, { 'TerraDCAT_ap:hasDataUsePermission': 'DUO:0000007' }))
+    const { getByText } = render(makeDatasetReleasePolicyDisplayInformation('DUO:0000007'))
     expect(getByText('Disease specific research')).toBeTruthy()
   })
 
   it('uses unspecified as the data use policy if undefined', () => {
-    const { getByText } = render(h(DatasetReleasePolicyDisplayInformation, { 'TerraDCAT_ap:hasDataUsePermission': undefined }))
+    const { getByText } = render(makeDatasetReleasePolicyDisplayInformation(undefined))
     expect(getByText('Unspecified')).toBeTruthy()
   })
 
   it('uses given data use policy as the data use policy if unknown', () => {
-    const { getByText } = render(h(DatasetReleasePolicyDisplayInformation, { 'TerraDCAT_ap:hasDataUsePermission': 'Something else' }))
+    const { getByText } = render(makeDatasetReleasePolicyDisplayInformation('Something else'))
     expect(getByText('Something else')).toBeTruthy()
   })
+
+  it('generates consortium titles properly', () => {
```

Comment:
These tests could mostly get pulled into a separate PR if we wanted to make this smaller.

Reply:
I think it's worth keeping them in here since we have coverage on each PR now.